src/main/resources/application.yml

```
@@ -120,6 +120,10 @@ spring:
       - task-queue: CreateAssessmentsWF
         activity-beans:
           - assessmentsCreationActivityImpl
+      # export file expiration handler
```

let's keep more generic this section in order to set here also the future WF to handle exportFile
```suggestion
      # export file
```